client/jest.config.js

```diff
@@ -0,0 +1,52 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // Automatically clear mock calls and instances between every test
+  clearMocks: true,
+
+  // An array of glob patterns indicating a set of files for which coverage information should be collected
+  collectCoverageFrom: [
+    '**/src/components/**/*.{js}',
+    '!**/src/components/**/*{mock,stories,style,index}.{js}',
+    '!**/node_modules/**',
+    '!**/public/**',
+  ],
+
+  // The directory where Jest should output its coverage files
+  coverageDirectory: '<rootDir>/reports/coverage',
```

Comment:
Same path should be added in package.json file, `npm test:clean` script. Also, the same should be added in .gitignore.